WordPress/Classes/ViewRelated/Gutenberg/EditorMediaUtility.swift

```diff
@@ -140,6 +145,7 @@ class EditorMediaUtility {
         let imageDownload = AuthenticatedImageDownload(
             url: requestURL,
             blog: post.blog,
+            callbackQueue: .main,
```

Comment:
This should be an improvement to the existing behaviour, since `onFailure` wasn't called on the main thread, but I believe this function requires the error handling to be on the main thread.